lib/kerror/codes/7-security.json

```diff
@@ -45,6 +45,12 @@
           "code": 7,
           "message": "Error verifying token: %s",
           "class": "InternalError"
+        },
+        "refresh_forbidden": {
+          "description": "The provided token cannot be refreshed.",
+          "code": 8,
+          "message": "Token cannot be refreshed.",
```

Comment:
We should provide the reason to the user so he can know if he is using an API key or a token with infinite TTL